components/PokemonListItem/pokemonListItem.module.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  padding: 1rem;
+  background-color: #2adab1;
```

Comment:
No hard-coding colors, px values or paddings - everything should be theme or reusable classes driven - that's how you strive for consistency

Reply:
Done as stated